optimum/intel/openvino/configuration.py

```diff
@@ -624,6 +634,7 @@ def to_nncf_dict(self) -> Dict[str, Any]:
             "gptq": self.gptq,
             "lora_correction": self.lora_correction,
             "backup_mode": backup_mode,
+            **self.init_kwargs,
```

Comment:
Why do you need 2 different names `init_kwargs` and `kwargs`?

Reply:
My reasoning was not to create a property named `kwargs` at `OVQuantizationConfigBase` to avoid confusion as name "kwargs" is commonly used to refer to additional keyword arguments not listed in a method signature. That's why I chose `init_kwargs` to convey the meaning that these are the keyword arguments which were provided at config initialization.

Reply:
Renamed